### DISCUSSION TITLE
Is it possible to use Reactome terms of mouse for the search?

### DISCUSSION THREAD

**Discussion by fengx90** [egeulgen/pathfindR#168] [Q&A]
Dear Ege,

The "get_gene_sets_list" function only retrieve human pathways when using Reactome as source (as indicated in the function https://reactome.org/download/current/ReactomePathways.gmt.zip).

Is there any way to use Reactome pathways annotated to mouse? Thanks very much and looking forward to your answer.

Best,
Feng

**Comment by egeulgen** [comment]
The function `get_gene_sets_list()` only supports retrieval of H.sapiens gene sets from Reactome. You may manually obtain the M.musculus Reactome gene sets and follow [this vignette](https://egeulgen.github.io/pathfindR/articles/non_hs_analysis.html) for non-hs analysis.
Hope this helps,
-E
